src/VAC/View.cpp

```diff
@@ -76,6 +76,7 @@ const constexpr auto POLYGON_ARROUND_VERTICES_FROM = 4;
 const constexpr auto POLYGON_ARROUND_VERTICES_TO = 9;
 const constexpr auto POLYGON_ARROUND_ALPHA = 0;
 const constexpr auto POLYGON_ARROUND_LINE_SIZE = 0.5;
+const constexpr auto IS_DRAW_CIRCLES_AS_CURVES = false;
```

Comment:
IS_/is prefix for getters only, at in those places a question i.e. "isVisible(?)", but constants are statements or facts, so no IS_ there.

Reply:
done